src/main/java/frc5124/robot2020/RobotContainer.java

```diff
@@ -24,16 +24,23 @@
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
-import edu.wpi.first.wpilibj.shuffleboard.SimpleWidget;
+import edu.wpi.first.wpilibj2.command.Command;
 
 import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.button.POVButton;
 
-import frc5124.robot2020.commands.*;
```

Comment:
why do we need to include every command separately in master code?

Reply:
commands* doesn't import folders. folders exist for organization of command code